src/test/java/seedu/guest/storage/JsonGuestBookStorageTest.java

```diff
@@ -42,59 +42,59 @@ private Path addToTestDataPathIfNotNull(String prefsFileInTestDataFolder) {
 
     @Test
     public void read_missingFile_emptyResult() throws Exception {
-        assertFalse(readAddressBook("NonExistentFile.json").isPresent());
+        assertFalse(readGuestBook("NonExistentFile.json").isPresent());
     }
 
     @Test
     public void read_notJsonFormat_exceptionThrown() {
-        assertThrows(DataConversionException.class, () -> readAddressBook("notJsonFormatGuestBook.json"));
+        assertThrows(DataConversionException.class, () -> readGuestBook("notJsonFormatGuestBook.json"));
     }
 
     @Test
-    public void readAddressBook_invalidPersonAddressBook_throwDataConversionException() {
-        assertThrows(DataConversionException.class, () -> readAddressBook("invalidGuestGuestBook.json"));
+    public void readGuestBook_invalidGuestBook_throwDataConversionException() {
```

Comment:
Should this be readGuestBook_invalidGuestGuestBook_throwDataConversionException instead of readGuestBook_invalidGuestBook_throwDataConversionException?